app/views/admin/configs/show.html.erb

```diff
@@ -256,136 +254,60 @@
+                        <%= inline_svg_tag("#{provider.provider_name}.svg", class: "crayons-icon", aria: true, title: "#{provider.official_name} logo") %>
+                      </figure>
+                      <div class="config-authentication__item--container">
+                        <div
+                          class="config-authentication__item mb-0 flex items-baseline justify-between <%= auth_provider_enable_btn_tooltip_class %>"
+                          data-tooltip="<%= enable_disable_btn_tooltip_text %>">
+                          <div>
+                            <p class="config-authentication__item--label fw-medium <%= authentication_enabled_providers.include?(provider) ? "pb-0" : "" %>">
+                              <%= provider.official_name %>
                             </p>
-                          <%= f.text_field :facebook_secret,
-                                           class: "crayons-textfield",
-                                           value: SiteConfig.facebook_secret,
-                                           placeholder: Constants::SiteConfig::DETAILS[:facebook_secret][:placeholder] %>
+                            <% if authentication_enabled_providers.include?(provider) %>
+                              <div class="mb-5 flex items-center">
+                                <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon admin-config-checkmark", title: "Checkmark") %>
+                                <small class="crayons-field__description ml-1">Enabled</small>
+                              </div>
+                            <% end %>
+                          </div>
+                          <button
+                            class="crayons-btn crayons-btn--secondary"
+                            id="<%= provider.provider_name %>-auth-btn"
+                            data-button-text="<%= authentication_enabled_providers.include?(provider) ? "edit" : "enable" %>"
+                            data-action="click->config#enableOrEditAuthProvider"
+                            data-auth-provider-enable="<%= provider.provider_name %>"
+                            data-enable-auth="<%= authentication_enabled_providers.include?(provider) ? "true" : "false" %>"
+                            <%= auth_provider_enable_btn_disable_class %>>
+                            <%= authentication_enabled_providers.include?(provider) ? "Edit" : "Enable" %>
+                          </button>
                         </div>
-                      </fieldset>
-                    </div>
-                  </div>
+                        <div id="<%= provider.provider_name %>-auth-settings" class="config-authentication__item--body hidden">
+                          <fieldset class="config-authentication-form">
+                            <%= render partial: "auth_provider_settings",
+                                       locals: {
+                                         f: f,
```

Comment:
f?!?!

Reply:
@joshpuetz You mean rename `auth_provider_settings` file?

Reply:
No, I mean the local variable `f`: what is it?

Reply:
Oh! 😆  It's the form! 🤣

Reply:
Ah, true, f is used for 'form' a lot in Rails. I still feel like naming it could be helpful, curious what others think. Definitely not something that would block my approval!

Reply:
I'm keeping it as `f` 😄  @vaidehijoshi